api/controllers/EventController.js

```diff
@@ -135,7 +136,10 @@ var EventController = {
                                 coordinates : [lng, lat] },
                             $maxDistance : radius
                         }
-                    }
+                    },
+                    $and:[
+                        { $or : [{title:regex}, {description:regex}]}
```

Comment:
would that be ok if `search` is not passed? so `regex` will be `undefined`

Reply:
Yeah, I have already test this case. 
It will ignore the "and" and "or" clause if search is undefined.

Reply:
👍